ci/coverity_unsuccessful.sh

```diff
@@ -9,7 +9,7 @@ rm -f coverity/*
 mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 ci_envs="$mydir/parse_ci_envs.sh"
 if [ -e "${ci_envs}" ]; then
-  # shellcheck disable=SC1091
+  # shellcheck source=parse_ci_envs.sh
```

Comment:
(lint) Couldn't parse this shellcheck annotation. [SC1073]
(lint) Expected "disable". Fix any mentioned problems and try again. [SC1072]